src/utils.ts

```diff
@@ -63,15 +64,17 @@ export function execute(
   return new Promise((resolve, reject) => {
     let timer: number;
     const req = https.get(url, (resp) => {
+      const decoder = new StringDecoder("utf8");
```

Comment:
Simply adding `resp.setEncoding("utf8");` here should do the job, what do you think?

[Documentation](https://nodejs.org/docs/latest-v7.x/api/stream.html#stream_readable_setencoding_encoding)

Reply:
That's more like it, thanks @zyc9012!